blueprints/app/files/config/targets.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-env node */
+
+module.exports = {
+  browsers: [] // ["last 2 versions", "safari >= 7"]
```

Comment:
Let's inline the default here also (instead of the comment). This lets us update the default for new apps without potentially breaking existing ones.